lib/solargraph/parser/rubyvm/class_methods.rb

```diff
@@ -22,7 +21,6 @@ def parse_with_comments code, filename = nil
         # @param filename [String, nil]
         # @param line [Integer]
         # @return [Parser::AST::Node]
-        # @sg-ignore
```

Comment:
One of these days I'd like to look into solargraph warning when an @sg-ignore is not needed, so the user makes sure to remove them once the issue is addressed.